packages/wallets/src/features/cfd/screens/DynamicLeverage/DynamicLeverageScreen.tsx

```diff
@@ -7,9 +7,11 @@ import { DynamicLeverageMarketCard } from './DynamicLeverageMarketCard';
 import './DynamicLeverageScreen.scss';
 
 const DynamicLeverageScreen = () => {
-    const { data: dynamicLeverages } = useDynamicLeverage();
+    const { data: dynamicLeverages } = useDynamicLeverage('mt5');
```

Comment:
Not covering any loading state ? 🤔

Reply:
is there no error state expected as well?

Reply:
This screen is rendered but hidden behind the jurisdiction modal only visible when use clicks the dynamic leverage button so I don't think the loading state will be visible.
We aren't handling the error state for now.